index.css

```diff
@@ -0,0 +1,81 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    background: rgb(233, 230, 226);
```

Comment:
Consider using css variables to store repeating properties such as colors checkout this resource https://www.w3schools.com/css/css3_variables.asp